video/out/filter_kernels.c

```diff
@@ -54,25 +55,26 @@ const struct filter_kernel *mp_find_filter_kernel(const char *name)
 }
 
 // sizes = sorted list of available filter sizes, terminated with size 0
-// inv_scale = source_size / dest_size
+// filter_scale = source_size / dest_size
 bool mp_init_filter(struct filter_kernel *filter, const int *sizes,
-                    double inv_scale)
+                    double filter_scale)
```

Comment:
To clarify: I would keep the parameter named “inv_scale”, since it comes from the actual scale factor. “inv_scale” turns into “filter_scale” by clamping it to values above 1.